GraphQL/JSON+Helper.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  JSON+Helper.swift
+//  GraphQL
+//
+//  Created by YOU-HSUAN YU on 2021/9/7.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import Foundation
+import Apollo
+
+public typealias JSON = [Any]
```

Comment:
Why the JSON is an array of Any instead of the dictionary `[String: String]` type? 
As your comment below, wouldn't it be much more straightforward that the SDK claims `seatInfo` as the type `[String: String]` than a custom object type?
> SeatInfo to be a dictionary type ([String: String]).